sysboundary/sysboundary.cpp

```diff
@@ -76,10 +76,13 @@ SysBoundary::~SysBoundary() {
  * help.
  */
 void SysBoundary::addParameters() {
-   Readparameters::addComposing("boundaries.boundary", "List of boundary condition (BC) types to be used. Each boundary condition to be used has to be on a new line boundary = YYY. Available (20140113) are Outflow Ionosphere Maxwellian.");
-   Readparameters::add("boundaries.periodic_x","If 'yes' the grid is periodic in x-direction. Defaults to 'no'.","no");
-   Readparameters::add("boundaries.periodic_y","If 'yes' the grid is periodic in y-direction. Defaults to 'no'.","no");
-   Readparameters::add("boundaries.periodic_z","If 'yes' the grid is periodic in z-direction. Defaults to 'no'.","no");
+   Readparameters::addComposing(
+       "boundaries.boundary",
+       "List of boundary condition (BC) types to be used. Each boundary condition to be used has to be on a new line "
+       "boundary = YYY. Available options are: Outflow, Ionosphere, Maxwellian, User.");
+   Readparameters::add("boundaries.periodic_x", "Set the grid periodicity in x-direction. true(default)/false.", true);
+   Readparameters::add("boundaries.periodic_y", "Set the grid periodicity in y-direction. true(default)/false.", true);
+   Readparameters::add("boundaries.periodic_z", "Set the grid periodicity in z-direction. true(default)/false.", true);
```

Comment:
I only now (when trying to bring another branch up-to-date and finding conflicts) see that here the default value of periodicity has changed. 

Although it can make sense (e.g. if no other boundary has been set, then assume periodicity) it should have been documented in the PR. If some other boundary type is set, does that now automatically unset periodicity, or can there be a clash?